CONTRIBUTING.md

```diff
@@ -65,6 +65,7 @@ Please contribute! Meshery documentation uses GitHub Pages to host the docs site
 `vi <specific page>.md`
 1. Run site locally to preview changes.
 `make site`
+* **Note:** *From the Makefile, this command is actually running `$ bundle exec jekyll serve --drafts --livereload`. There are two Jekyll configuration, `jekyll serve` for developing locally and `jekyll build` when deploying for production based on differences of the handling of trailing slashes between Jekyll and GitHub Pages.*
```

Comment:
This line is not appropriate, jekyll build is used by production parsers which simply generates the output of jekyll serve in simple html+js. Mentioning about jekyll build as production server is completely wrong.
I won't recommend such controversial statement to be added in the contributing file, else the below added description for WSL is great.